javascript/javascript1/week2/classwork/count-down.js

```diff
@@ -0,0 +1,15 @@
+function countDown(n) {
+  for (i = n; i >= 0; i--) {
+    if (i === 10) {
+      console.log("OMG it started");
+    } else if (i !== 0 && i !== 10) {
+      console.log(i);
+    } else if (i === 0) {
+      console.log("Happy new year!");
+    } else {
+      console.log("it is not a new year");
+    }
+  }
+}
+
+countDown(10);
```

Comment:
I like your implementation, maybe we can discuss some improvements!

Reply:
Sure Mutaq.. l would like to know what can be improved. Let me know when you have time.